site/content/docs/5.2/components/list-group.md

```diff
@@ -311,15 +317,15 @@ As part of Bootstrap's evolving CSS variables approach, list groups now use loca
 
 {{< scss-docs name="list-group-variables" file="scss/_variables.scss" >}}
 
-### Mixins
+### Sass mixins
```

Comment:
IDK if it must be done in this PR or outside but we have inconsistencies in our components for these sections:
- "Sass mixins" / "Sass mixin" / "Mixins"
- "Sass loops" / "Sass loop" / "Loop"

Reply:
Yeah, I'll leave this comment unresolved and make that change for a follow-up. Think it needs the `Sass ` in front given it's under the CSS section.